charts/sentinel-tick/templates/deployment.yaml

```diff
@@ -3,20 +3,29 @@ kind: Deployment
 metadata:
   name: {{ include "sentinel-tick.fullname" . }}
   labels:
-    {{- include "sentinel-tick.labels" . | nindent 4 }}
+    app: sentinel-tick
+    suite: sentinel
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
```

Comment:
I'm unsure of proper patterns here and it might be worth getting input from maybe @travisperson or @ognots on this. That said, `sentinel-tick.labels` and `selectorLabels` have some inclusions which are now missing from this change. (See https://github.com/filecoin-project/helm-charts/blob/master/charts/sentinel-tick/templates/_helpers.tpl#L36-L51.) I suspect that this metadata may have some value that I'm unaware of.

Reply:
Leaving as is and merging because I don't know.